packages/react-components/src/components/Button/Button.module.scss

```diff
@@ -278,4 +278,29 @@ $base-class: 'btn';
       margin: 0;
     }
   }
+
+  &--subtle {
```

Comment:
Instead of putting the styles at the very end I would consider some type-based order (eg. similar as in Storybook). Most likely somewhere around lines 202-204